testing/test_full.py

```diff
@@ -425,6 +425,45 @@ def test_inference_classification_rad_3d(device):
     print("passed")
 
 
+def test_inference_classification_with_logits_rad_3d(device):
+    # read and initialize parameters for specific data dimension
+    parameters = parseConfig(
+        testingDir + "/config_classification.yaml", version_check=False
+    )
+    parameters["modality"] = "rad"
+    parameters["patch_size"] = patch_size["3D"]
+    parameters["model"]["dimension"] = 3
+    parameters["model"]["amp"] = True
+    parameters["model"]["final_layer"] = "logits"
+    parameters["nested_training"]["validation"] = 2
```

Comment:
Need to ensure that the inference works when even a single validation fold is presented.

Reply:
both single and multi fold cases should be covered with the latest commit